mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -74,11 +88,17 @@ impl Bridge {
             .into_iter()
             .map(|topic| topic.try_into())
             .collect::<Result<Vec<_>, _>>()?;
+
+        // component_sender can be used for configuration changes
+        let (connectivity_sender, connectivity_receiver) =
```

Comment:
I wonder if we can move channel creation inside pump. wrap sender part into PumpHandle and pass PumpHandle everywhere instead? Pattern is similar what we have in the Broker struct